src/modules/resource/topo.c

```diff
@@ -87,26 +87,35 @@ static int drain_self (struct topo *topo, const char *reason)
 static int topo_verify (struct topo *topo, json_t *R, bool nodrain)
```

Comment:
It might be helpful later to have a comment block here noting that gpus are purposefully not verified with a reference to #4181?

Reply:
Yes, good call! I'll add that.